Usage of non-standard explicit type conversion syntax

https://github.com/strukturag/libheif/blob/d6e233595ca283eecd649d74f7b35762957ec4a6/libheif/plugins/encoder_svt.cc#L675

error C4576: a parenthesized type followed by an initializer list is a non-standard explicit type conversion syntax

This seems to be throwing an error with Visual C++ 2022.

I hit the error trying to enable SVT-AV1 in https://github.com/conda-forge/libheif-feedstock/pull/9

This will also occur with clang and some flags

```
/home/bradh/coding/libheif/libheif/plugins/encoder_svt.cc:677:32: error: compound literals are a C99-specific feature [-Werror,-Wc99-extensions]
  svt_config.rc_stats_buffer = (SvtAv1FixedBuf) {nullptr, 0};
                               ^
1 error generated.
make[2]: *** [libheif/CMakeFiles/heif.dir/build.make:594: libheif/CMakeFiles/heif.dir/plugins/encoder_svt.cc.o] Error 1
make[1]: *** [CMakeFiles/Makefile2:1163: libheif/CMakeFiles/heif.dir/all] Error 2
make: *** [Makefile:166: all] Error 2
```

The fix looks fairly simple, will send a PR.

--- a/libheif/plugins/encoder_svt.cc
+++ b/libheif/plugins/encoder_svt.cc
@@ -674,7 +674,7 @@ struct heif_error svt_encode_image(void* encoder_raw, const struct heif_image* i
   svt_config.logical_processors = encoder->threads;
 
   // disable 2-pass
-  svt_config.rc_stats_buffer = (SvtAv1FixedBuf) {nullptr, 0};
+  svt_config.rc_stats_buffer = SvtAv1FixedBuf {nullptr, 0};
 
   svt_config.rate_control_mode = 0; // constant rate factor
   //svt_config.enable_adaptive_quantization = 0;   // 2 is CRF (the default), 0 would be CQP

